public/index.html

```diff
@@ -25,14 +25,14 @@
       <h1 class="header__primary">
         <span class="header__primary--top hdr-text">A Viget Exploration</span>
         <img class="header__primary--btm" src="assets/images/title.svg" alt="improving medical adherence">
+        <img class="header__primary--pills1" src="assets/images/pills.svg" alt="">
+        <img class="header__primary--pills2" src="assets/images/yellow-pill.svg" alt="">
+        <img class="header__primary--pills3" src="assets/images/more-pills.svg" alt="">
```

Comment:
these `<img>` being in the `<h1>` feels a little funny...  I think the first `<img>` with the alt text is good - you need that text in your `<h1>`. But the other ones? I don't really have a good reason for this though... just preference and perhaps some accessibility paranoia (I don't know if a random screen reader will have a problem with the heading due to the images).